### PR TITLE
Misc 3.3 docs changes

### LINKED ISSUE #13330: Foward-port 3.2.2 SRI hashes and changelog
This is an issue to remember to port these assets to `branch-3.3` before release. Additionally, the release automation should add a step to submit this reminder issue automatically on release. I am not sure we can fully automate the forward port itself, but it's not a lot of  work to do (mostly it's just easy to forget).

### LINKED ISSUE #13360: Explicitly document conf.py changes for bokeh.sphinxext
Ref: https://discourse.bokeh.org/t/how-to-use-bokeh-sphinxext/10805/1

Given the confusion there, we could add an explicit statement along the lines of 
> To use this extension, add `"bokeh.sphinxext.bokeh_plot"` to the `extensions` list in your Sphinx configuration module. 

At the very least for `bokeh_plot` and maybe a few other of the more possibly user-facing extensions, but potentially to all, for the sake of consistency.

### PR DESCRIPTION
**Pull request by bryevdv**
- [ ] issues: fixes #13330 
- [ ] issues: fixes #13360 

Note our version switcher list was getting pretty long so I removed some of the older versions.

### CODE REVIEW

File: docs/CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+    - #13305 [component: docs] 13276 easy fixes in documentation
+    - #13326 Backports for 3.2.2
+
+2023-07-20    3.2.1:
+--------------------
+  * bugfixes:
+    - #13139 [component: bokehjs] Error in downstream library when rendering WebGL canvas with width or height of zero
+    - #13162 [component: docs] Broken link in docs/first_steps/first_steps_9.html
+    - #13248 [component: bokehjs] [BUG] Browser freezes when deleting notebook cell containing plot with TileSource
+    - #13252 [component: bokehjs] [BUG]  plots in png generated by SaveTool on gridplot layout overlap
```
Comment (ianthomas23) [inline docs/CHANGELOG:20]:
```suggestion
    - #13252 [component: bokehjs] [BUG] plots in png generated by SaveTool on gridplot layout overlap
```

File: docs/CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+    - #13255 [component: bokehjs] [BUG] Enabling/Disabled DatePickers does not work
+    - #13262 [component: bokehjs] [BUG] Legend draws 3 columns but `ncols` is set to 2
+    - #13254 [component: bokehjs] Use `../core/kinds` instead of `core/kinds` in imports
+    - #13270 [component: docs] Update theme.py
+    - #13274 [component: docs] fix unterminated string literals in example code [skip ci]
```
Comment (ianthomas23) [inline docs/CHANGELOG:25]:
```suggestion
    - #13274 [component: docs] fix unterminated string literals in example code
```

**Review by ianthomas23 — APPROVED** [review]
Just a couple of nits.

File: docs/CHANGELOG (hunk 1 above)
Comment (bryevdv) [reply docs/CHANGELOG:20]: doh, I saw the approval and merged on autopilot. That said, changelogs are auto-generated on release, so updating them manually is maybe not worth it in any case.

**Pull request merged**